src/api.js

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+
+const ncNews = "https://oscar-northcoders-news.onrender.com";
+
+export const getArticles = () => {
+  const articlesURL = ncNews + "/api/articles";
+  return axios.get(articlesURL).then((response) => {
+    return response.data?.articles;
+  });
+};
+
+export const getSingleArticle = (articleId) => {
+  const singleArticleURL = `${ncNews}/api/articles/${articleId}`;
```

Comment:
It's generally desirable to keep our code consistent... Above you use concatenation and here we use template literals. I would pick one method and stick to it